.github/ISSUE_TEMPLATE/RFC.md

```diff
@@ -1,7 +1,6 @@
 ---
 name: RFC for new interface
-about: Use this template to request new functionality or change behavior of the
-library
+about: Use this template to request new functionality or change behavior of the library
```

Comment:
"change **the** behavior of the library"?

Reply:
fixed